### PR TITLE
Issue-841 - Add documentation for deb distribution

### PR DESCRIPTION
**Pull request by TheCakeIsNaOH**
Documentation for https://github.com/strongbox/strongbox/pull/1612

### CODE REVIEW

File: docs/user-guide/getting-started.md [hunk 1]
```diff
@@ -68,6 +68,34 @@ sudo service strongbox start
 
 ```
 
+
+```linuxdeb linenums="1" tab="Linux (deb)"
+# Open a terminal
+
+# First, make sure you have a JRE with version 1.8 or greater
```
Comment (steve-todorov) [inline docs/user-guide/getting-started.md]: We do not support JRE higher than 1.8 at the moment :)
```suggestion
# First, make sure you have a JRE with version 1.8
```

Comment (TheCakeIsNaOH) [reply docs/user-guide/getting-started.md]: Done, also adjusted the RPM as I based the deb off of it.

**Review by steve-todorov — APPROVED** [review]
LGTM! Thanks! :)

**Comment by carlspring** [comment]
Thank you very much!

**Pull request merged**